src/components/CommentInputBox.jsx

```diff
@@ -0,0 +1,59 @@
+import { postComment } from '../api';
+import { useState, useContext } from 'react';
+import { UserContext } from '../contexts/LoggedInUser';
+
+export default function CommentInputBox({ refreshComments, article_id }) {
+
+    const [ commentBody, setComment ] = useState('');
+    const { submitComment, isLoading, error } = sendComment(refreshComments, article_id);
+    const { user } = useContext(UserContext);
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        submitComment(user, commentBody);
+        setComment('');
+    }
+
+    if (isLoading) return <div className='comment-input-box'>Loading...</div>;
+    if (error) return <div className='comment-input-box'>Error: {error.msg}</div>;
```

Comment:
Nice use of error fetching component, though the `error.msg` property sometimes doesn't exist (like when there's no internet connection) so you might want to use a more generic error message